qmk setup fails on Fedora 36

`qmk setup` fails on a fresh Fedora 36 installation with `usb-libusb.c:28:10: fatal error: usb.h: No such file or directory`.

Manually installing `libusb-devel` before running `qmk setup` fixes the error.

```
--2022-09-01 20:12:31--  https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz
Resolving www.obdev.at (www.obdev.at)... 78.46.114.187
Connecting to www.obdev.at (www.obdev.at)|78.46.114.187|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://www.obdev.at/ftp/pub/Products/vusb/bootloadHID.2012-12-08.tar.gz [following]
--2022-09-01 20:12:31--  https://www.obdev.at/ftp/pub/Products/vusb/bootloadHID.2012-12-08.tar.gz
Reusing existing connection to www.obdev.at:443.
HTTP request sent, awaiting response... 200 OK
Length: 107970 (105K) [application/x-gzip]
Saving to: ‘STDOUT’

-                                                           100%[========================================================================================================================================>] 105.44K  --.-KB/s    in 0.08s   

2022-09-01 20:12:31 (1.26 MB/s) - written to stdout [107970/107970]

gcc  -O2 -Wall `libusb-config --cflags` -c main.c -o main.o
/bin/sh: line 1: libusb-config: command not found
gcc  -O2 -Wall `libusb-config --cflags` -c usbcalls.c -o usbcalls.o
/bin/sh: line 1: libusb-config: command not found
In file included from usbcalls.c:19:
usb-libusb.c:28:10: fatal error: usb.h: No such file or directory
   28 | #include <usb.h>
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:46: usbcalls.o] Error 1
```

While a failure, isnt considered a hard failure within `qmk setup`. This would only be an issue if you need to use the `bootloadHID` tool for flashing.

--- a/util/install/fedora.sh
+++ b/util/install/fedora.sh
@@ -8,7 +8,7 @@ _qmk_install() {
         clang diffutils git gcc glibc-headers kernel-devel kernel-headers \
         make unzip wget zip python3 avr-binutils avr-gcc avr-libc \
         arm-none-eabi-binutils-cs arm-none-eabi-gcc-cs arm-none-eabi-newlib \
-        avrdude dfu-programmer dfu-util hidapi libusb1-devel
+        avrdude dfu-programmer dfu-util hidapi libusb-devel
 
     python3 -m pip install --user -r $QMK_FIRMWARE_DIR/requirements.txt
 }

